benchmarks/setup/README.md

```diff
@@ -71,3 +71,23 @@ In order to do this easily, just run:
 ```
 
 This will switch to the default namespace, delete the given namespace and delete the corresponding folder.
+
+## How to run a Benchmark in Camunda Cloud
+
+We currently waiting until https://github.com/camunda-cloud/enhancements/blob/master/CEP017-programmatic-access-to-console.md is implemented, until this is done we need to do some steps manually.
+
+### Setup Cloud Cluster
+
+ * Go to https://console.cloud.ultrawombat.com
+ * Login with your credentials, reach out to cloud team if you haven't one
+ * Create a new cluster, as region choose 'new chaos'. This means the cluster will be created in 'ultrachaos' gke cluster
```

Comment:
Who is this document targeted at? I guess customers/community members don't have access to this region. So I guess this is targeted internally. I think it makes sense to make this more explicit.

Reply:
I can add something like this:


_This targets only internal use cases and is not available for the outside._

Reply:
@pihme like this?

Reply:
Yes, but maybe rephrase _outside_ to _external parties_

Reply:
ok thanks